### PR TITLE
Cherrypick recent CLI changes (1)

### PR DESCRIPTION
**Pull request by meiliang86**
Cherrypick the following two changes:
https://github.com/temporalio/temporal/commit/8c17ae8cdf174b8fa3e16df4a90007d58cb951e0
https://github.com/temporalio/temporal/commit/2fa70b3be0cdb7106c1b3d9fd399e779b865ccee

### CODE REVIEW

File: tools/cli/admin.go [hunk 1]
```diff
@@ -113,6 +113,26 @@ func newAdminWorkflowCommands() []cli.Command {
...
+			Flags: append(
+				getDBFlags(),
+				cli.IntFlag{
+					Name:  FlagShardID,
+					Usage: "The Id of the shard to describe",
+				},
+				cli.StringFlag{
+					Name:  FlagTargetCluster,
+					Value: "active",
+					Usage: "Temporal cluster to use",
```
Comment (mantas-sidlauskas) [inline tools/cli/admin.go:129]: Misleading "Usage" description

File: tools/cli/admin.go [hunk 2]
```diff
@@ -876,6 +902,7 @@ func getDBFlags() []cli.Flag {
 		},
 		cli.StringFlag{
 			Name:  FlagKeyspace,
+			Value: "temporal",
```
Comment (mantas-sidlauskas) [inline tools/cli/admin.go:905]: should be "cadence"

Comment (meiliang86) [reply tools/cli/admin.go]: Updated.

File: tools/cli/admin.go (hunk 1 above)
Comment (meiliang86) [reply tools/cli/admin.go]: Updated.

**Pull request merged**